src/App.tsx

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const selectItems = [3, 5, 10, 20];
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
```

Comment:
It's a good practice to name state setters consistently with the state variable. For `currentPage`, the setter should be `setCurrentPage`, which is correctly named, but for `perPage`, consider renaming the setter to `setItemsPerPage` for better readability.